### ISSUE TITLE
Add graphs

### ISSUE THREAD

**Issue by D4nte** [comit-network/comit.network#13]
## Problem

It'd be nice to have graphs in the documentation such as:
- Network graph for `start-env` in the Getting Started section #10

However, plantuml does not allow to do "pretty" graph and it can be quite a time sink.

## Goal

Clear documentation and professional looking graphs.

## Recommendation

A current position for a _product designer_ is open, is that something that could be part of his role?

**Comment by D4nte** [comment]
@bonomat @da-kami please review and provide input.

**Comment by bonomat** [comment]
I agree, plantuml can be quite ugly. Additionally, I think an uml diagram is a bit of an overkill for the documentation. 

Depending on what you want to depict a simple google draw for the beginning should do it. We can always ask a designer to make it sexy later on.

**Comment by D4nte** [comment]
As it can easily be a time sink I'll leave it as a separate issue for now (will refine once #10 is merged)

**Comment by da-kami** [comment]
I think plantuml is sufficient for small diagrams. I don't find it ugly when setting the colours to black (see diagrams in RFCs).
In the end we write documenation for Devs, not end users and if diagrams help I would always put them there. 
Google draw is good for block/Venn diagrams, not uml. When it comes to uml plantuml is actually a great tool to create diagrams fast. Complex diagrams are, however, more difficult to maintain and display one one page (but that is more of a general uml problem than a plantuml problem).

**Comment by da-kami** [comment]
> As it can easily be a time sink I'll leave it as a separate issue for now (will refine once #10 is merged)

I would argue: if we sink time to understand something better and make it better in the long run, that's ok. If we sink time because of the tool that's not ok.

**Comment by D4nte** [comment]
> > As it can easily be a time sink I'll leave it as a separate issue for now (will refine once #10 is merged)
> 
> I would argue: if we sink time to understand something better and make it better in the long run, that's ok. If we sink time because of the tool that's not ok.

I am just scoping this timesink out of the ongoing issue.

**Comment by D4nte** [comment]
> I think plantuml is sufficient for small diagrams. I don't find it ugly when setting the colours to black (see diagrams in RFCs).
> In the end we write documenation for Devs, not end users and if diagrams help I would always put them there.
> Google draw is good for block/Venn diagrams, not uml. When it comes to uml plantuml is actually a great tool to create diagrams fast. Complex diagrams are, however, more difficult to maintain and display one one page (but that is more of a general uml problem than a plantuml problem).

I could not make a reasonable diagram to show what `start-env` boots with plantuml.

**Comment by D4nte** [comment]
In this specific instance I'd agree with @da-kami to re-use https://docs.google.com/presentation/d/1gNolSxwIKVU4sXOYH9yCzCpsdlil__4CpqKJFhNcllY/edit?ts=5e38c5ea&skip_itp2_check=true&pli=1#slide=id.g63e57de19a_0_184 and maybe simplify it.
